README.md

```diff
@@ -1 +1,88 @@
-# Login-page
\ No newline at end of file
+
```

Comment:
There are git conflicts in this file. Please remove those and only commit the required changes